[BUG] mysql cluster snapshot restore Failed

```
kbcli version
Kubernetes: v1.23.8+k3s1
KubeBlocks: 0.6.0-alph
```
```
create component resource constraint kb-resource-constraint-custom
apiVersion: apps.kubeblocks.io/v1alpha1
kind: ComponentResourceConstraint
metadata:
  annotations:
    meta.helm.sh/release-name: kubeblocks
    meta.helm.sh/release-namespace: default
  labels:
    app.kubernetes.io/managed-by: Helm
    resourceconstraint.kubeblocks.io/provider: kubeblocks
  name: kb-resource-constraint-custom
spec:
  constraints:
  - cpu:
      max: 2
      min: "0.1"
      step: "0.1"
    memory:
      sizePerCPU: 1Gi
  - cpu:
      max: 2
      min: "0.1"
      step: "0.1"
    memory:
      sizePerCPU: 2Gi
  - cpu:
      max: 2
      min: "0.1"
      step: "0.1"
    memory:
      sizePerCPU: 3Gi
  - cpu:
      max: 2
      min: "0.1"
      step: "0.1"
    memory:
      sizePerCPU: 4Gi
  - cpu:
      max: 2
      min: "0.1"
      step: "0.1"
    memory:
      sizePerCPU: 5Gi
-e 
      `kubectl apply -f kb-resource-constraint-custom.yaml`
    
componentresourceconstraint.apps.kubeblocks.io/kb-resource-constraint-custom created
 `kbcli class create custom-0.1c0.5g --cluster-definition apecloud-mysql --type mysql         --cpu 0.1 --memory 0.5Gi --constraint kb-resource-constraint-custom --storage name=data,size=20Gi  --namespace default `
    
Successfully created class [custom-0.1c0.5g].
create 1 replica Halt apecloud-mysql cluster

 `kbcli cluster create ltest              --termination-policy=Halt              --monitor=false --enable-all-logs=false --cluster-definition=apecloud-mysql --set class=custom-0.1c0.5g,replicas=1,storage=20Gi  --namespace default `
    
Info: --cluster-version is not specified, ClusterVersion ac-mysql-8.0.30 is applied by default
Cluster ltest created

 `kbcli cluster backup ltest --type snapshot --namespace default `
    
Backup backup-default-ltest-20230529191516 created successfully, you can view the progress:
        kbcli cluster list-backups --name=backup-default-ltest-20230529191516 -n default
 `kbcli cluster restore ltest-backup --backup backup-default-ltest-20230529191516 --namespace default `
    
Cluster ltest-backup created
```
```
k get cluster
NAME           CLUSTER-DEFINITION   VERSION           TERMINATION-POLICY   STATUS    AGE
ltest          apecloud-mysql       ac-mysql-8.0.30   WipeOut              Running   28m
ltest-backup   apecloud-mysql       ac-mysql-8.0.30   WipeOut              Failed    14m

k describe cluster ltest-backup
Name:         ltest-backup
Namespace:    default
Labels:       clusterdefinition.kubeblocks.io/name=apecloud-mysql
              clusterversion.kubeblocks.io/name=ac-mysql-8.0.30
Annotations:  kubeblocks.io/reconcile: 2023-05-29T11:42:35.41196025Z
API Version:  apps.kubeblocks.io/v1alpha1
Kind:         Cluster
Metadata:
  Creation Timestamp:  2023-05-29T11:15:36Z
  Finalizers:
    cluster.kubeblocks.io/finalizer
  Generation:        1
  Resource Version:  6106
  UID:               04bbda95-4b0e-4fb4-a1ab-2c258e75816b
Spec:
  Affinity:
    Pod Anti Affinity:     Preferred
    Tenancy:               SharedNode
  Cluster Definition Ref:  apecloud-mysql
  Cluster Version Ref:     ac-mysql-8.0.30
  Component Specs:
    Class Def Ref:
      Class:            custom-0.1c0.5g
    Component Def Ref:  mysql
    Monitor:            true
    Name:               mysql
    No Create PDB:      false
    Replicas:           1
    Resources:
      Limits:
        Cpu:     100m
        Memory:  512Mi
      Requests:
        Cpu:               100m
        Memory:            512Mi
    Service Account Name:  kb-sa-ltest
    Volume Claim Templates:
      Name:  data
      Spec:
        Access Modes:
          ReadWriteOnce
        Resources:
          Requests:
            Storage:   21Gi
  Termination Policy:  WipeOut
Status:
  Cluster Def Generation:  2
  Components:
    Mysql:
      Consensus Set Status:
        Leader:
          Access Mode:  None
          Name:
          Pod:          Unknown
      Phase:            Failed
      Pods Ready:       false
  Conditions:
    Last Transition Time:  2023-05-29T11:15:36Z
    Message:               The operator has started the provisioning of Cluster: ltest-backup
    Observed Generation:   1
    Reason:                PreCheckSucceed
    Status:                True
    Type:                  ProvisioningStarted
    Last Transition Time:  2023-05-29T11:42:35Z
    Message:               Successfully applied for resources
    Observed Generation:   1
    Reason:                ApplyResourcesSucceed
    Status:                True
    Type:                  ApplyResources
    Last Transition Time:  2023-05-29T11:42:35Z
    Message:               pods are not ready in Components: [mysql], refer to related component message in Cluster.status.components
    Reason:                ReplicasNotReady
    Status:                False
    Type:                  ReplicasReady
    Last Transition Time:  2023-05-29T11:15:37Z
    Message:               pods are unavailable in Components: [mysql], refer to related component message in Cluster.status.components
    Reason:                ComponentsNotReady
    Status:                False
    Type:                  Ready
  Observed Generation:     1
  Phase:                   Failed
Events:
  Type     Reason                    Age                   From                Message
  ----     ------                    ----                  ----                -------
  Normal   ComponentPhaseTransition  29m                   cluster-controller  Create a new component
  Normal   PreCheckSucceed           29m                   cluster-controller  The operator has started the provisioning of Cluster: ltest-backup
  Normal   ComponentPhaseTransition  29m                   cluster-controller  Running: false, PodsReady: true, PodsTimedout: false
  Warning  Failed                    29m                   cluster-controller  Cluster: ltest-backup is Failed, check according to the components message
  Normal   AllReplicasReady          6m34s (x6 over 29m)   cluster-controller  all pods of components are ready, waiting for the probe detection successful
  Warning  BackOff                   2m48s (x8 over 17m)   event-controller    Pod ltest-backup-mysql-0: Back-off restarting failed container
  Warning  ApplyResourcesFailed      2m48s (x2 over 8m7s)  cluster-controller  Operation cannot be fulfilled on statefulsets.apps "ltest-backup-mysql": the object has been modified; please apply your changes to the latest version and try again
  Normal   ApplyResourcesSucceed     2m48s (x3 over 29m)   cluster-controller  Successfully applied for resources
  Warning  ReplicasNotReady          2m48s (x6 over 25m)   cluster-controller  pods are not ready in Components: [mysql], refer to related component message in Cluster.status.components
```
```
 k get pod
NAME                   READY   STATUS    RESTARTS        AGE
ltest-mysql-0          4/4     Running   1 (46m ago)     46m
ltest-backup-mysql-0   4/4     Running   6 (6m14s ago)   33m

k describe pod ltest-backup-mysql-0
Name:             ltest-backup-mysql-0
Namespace:        default
Priority:         0
Service Account:  kb-sa-ltest
Node:             k3d-kb-playground-server-0/172.24.0.3
Start Time:       Mon, 29 May 2023 19:15:41 +0800
Labels:           app.kubernetes.io/component=mysql
                  app.kubernetes.io/instance=ltest-backup
                  app.kubernetes.io/managed-by=kubeblocks
                  app.kubernetes.io/name=apecloud-mysql
                  app.kubernetes.io/version=ac-mysql-8.0.30
                  apps.kubeblocks.io/component-name=mysql
                  apps.kubeblocks.io/workload-type=Consensus
                  controller-revision-hash=ltest-backup-mysql-55c9975fdb
                  statefulset.kubernetes.io/pod-name=ltest-backup-mysql-0
Annotations:      apps.kubeblocks.io/component-replicas: 1
Status:           Running
IP:               10.42.0.41
IPs:
  IP:           10.42.0.41
Controlled By:  StatefulSet/ltest-backup-mysql
Containers:
  mysql:
    Container ID:  containerd://0d0c474385943d572fbb4a240b605e104baa3988a1e2dc57b4bf1b8084bb254a
    Image:         registry.cn-hangzhou.aliyuncs.com/apecloud/apecloud-mysql-server:8.0.30-5.alpha8.20230523.g3e93ae7.7
    Image ID:      registry.cn-hangzhou.aliyuncs.com/apecloud/apecloud-mysql-server@sha256:5e4974945c604b8f475d020399c1d2d279af615b150a9ed19fb235a7a79a1bcc
    Ports:         3306/TCP, 13306/TCP
    Host Ports:    0/TCP, 0/TCP
    Command:
      /scripts/setup.sh
    State:          Running
      Started:      Mon, 29 May 2023 19:45:26 +0800
    Last State:     Terminated
      Reason:       Error
      Exit Code:    1
      Started:      Mon, 29 May 2023 19:38:49 +0800
      Finished:     Mon, 29 May 2023 19:42:34 +0800
    Ready:          True
    Restart Count:  6
    Limits:
      cpu:     100m
      memory:  512Mi
    Requests:
      cpu:     100m
      memory:  512Mi
    Environment Variables from:
      ltest-backup-mysql-env  ConfigMap  Optional: false
    Environment:
      KB_POD_NAME:               ltest-backup-mysql-0 (v1:metadata.name)
      KB_POD_UID:                 (v1:metadata.uid)
      KB_NAMESPACE:              default (v1:metadata.namespace)
      KB_SA_NAME:                 (v1:spec.serviceAccountName)
      KB_NODENAME:                (v1:spec.nodeName)
      KB_HOST_IP:                 (v1:status.hostIP)
      KB_POD_IP:                  (v1:status.podIP)
      KB_POD_IPS:                 (v1:status.podIPs)
      KB_HOSTIP:                  (v1:status.hostIP)
      KB_PODIP:                   (v1:status.podIP)
      KB_PODIPS:                  (v1:status.podIPs)
      KB_CLUSTER_NAME:           ltest-backup
      KB_COMP_NAME:              mysql
      KB_CLUSTER_COMP_NAME:      ltest-backup-mysql
      KB_CLUSTER_UID_POSTFIX_8:  8e75816b
      KB_POD_FQDN:               $(KB_POD_NAME).$(KB_CLUSTER_COMP_NAME)-headless.$(KB_NAMESPACE).svc
      MYSQL_ROOT_HOST:           %
      MYSQL_ROOT_USER:           <set to the key 'username' in secret 'ltest-backup-conn-credential'>  Optional: false
      MYSQL_ROOT_PASSWORD:       <set to the key 'password' in secret 'ltest-backup-conn-credential'>  Optional: false
      MYSQL_DATABASE:            mydb
      MYSQL_USER:                u1
      MYSQL_PASSWORD:            u1
      CLUSTER_ID:                1
      CLUSTER_START_INDEX:       1
      REPLICATION_USER:          replicator
      REPLICATION_PASSWORD:
      MYSQL_TEMPLATE_CONFIG:
      MYSQL_CUSTOM_CONFIG:
      MYSQL_DYNAMIC_CONFIG:
      KB_EMBEDDED_WESQL:         1
    Mounts:
      /data/mysql from data (rw)
      /etc/annotations from annotations (rw)
      /opt/mysql from mysql-config (rw)
      /scripts from scripts (rw)
      /var/run/secrets/kubernetes.io/serviceaccount from kube-api-access-pdc48 (ro)
  metrics:
    Container ID:  containerd://0fbbcfa99485dc2bb47808922d7d27e35f10e8311a6d40e7b9434522d972ab00
    Image:         registry.cn-hangzhou.aliyuncs.com/apecloud/mysqld-exporter:0.14.1
    Image ID:      registry.cn-hangzhou.aliyuncs.com/apecloud/mysqld-exporter@sha256:8a394d7e722a3c6be7082485eafb4c4082af6f2caf49b30446472ffeedc2ea28
    Port:          9104/TCP
    Host Port:     0/TCP
    Command:
      /bin/mysqld_exporter
      --collect.info_schema.wesql_consensus
    State:          Running
      Started:      Mon, 29 May 2023 19:15:49 +0800
    Ready:          True
    Restart Count:  0
    Liveness:       http-get http://:http-metrics/ delay=5s timeout=5s period=10s #success=1 #failure=6
    Readiness:      http-get http://:http-metrics/ delay=5s timeout=5s period=10s #success=1 #failure=6
    Environment Variables from:
      ltest-backup-mysql-env  ConfigMap  Optional: false
    Environment:
      KB_POD_NAME:               ltest-backup-mysql-0 (v1:metadata.name)
      KB_POD_UID:                 (v1:metadata.uid)
      KB_NAMESPACE:              default (v1:metadata.namespace)
      KB_SA_NAME:                 (v1:spec.serviceAccountName)
      KB_NODENAME:                (v1:spec.nodeName)
      KB_HOST_IP:                 (v1:status.hostIP)
      KB_POD_IP:                  (v1:status.podIP)
      KB_POD_IPS:                 (v1:status.podIPs)
      KB_HOSTIP:                  (v1:status.hostIP)
      KB_PODIP:                   (v1:status.podIP)
      KB_PODIPS:                  (v1:status.podIPs)
      KB_CLUSTER_NAME:           ltest-backup
      KB_COMP_NAME:              mysql
      KB_CLUSTER_COMP_NAME:      ltest-backup-mysql
      KB_CLUSTER_UID_POSTFIX_8:  8e75816b
      KB_POD_FQDN:               $(KB_POD_NAME).$(KB_CLUSTER_COMP_NAME)-headless.$(KB_NAMESPACE).svc
      MYSQL_USER:                <set to the key 'username' in secret 'ltest-backup-conn-credential'>  Optional: false
      MYSQL_PASSWORD:            <set to the key 'password' in secret 'ltest-backup-conn-credential'>  Optional: false
      DATA_SOURCE_NAME:          $(MYSQL_USER):$(MYSQL_PASSWORD)@(localhost:3306)/
    Mounts:
      /var/run/secrets/kubernetes.io/serviceaccount from kube-api-access-pdc48 (ro)
  kb-checkrole:
    Container ID:  containerd://a3be1511bd96c5a1f172466bb0cfbfd099def2b4a434c34aa1011efe99225456
    Image:         registry.cn-hangzhou.aliyuncs.com/apecloud/kubeblocks-tools:0.6.0-alpha.5
    Image ID:      registry.cn-hangzhou.aliyuncs.com/apecloud/kubeblocks-tools@sha256:b7c3b927862f0c1cb327f8b4a8eaf62a5da06ffe30caade6913a3159940f63c9
    Ports:         3501/TCP, 50001/TCP
    Host Ports:    0/TCP, 0/TCP
    Command:
      probe
      --app-id
      batch-sdk
      --dapr-http-port
      3501
      --dapr-grpc-port
      50001
      --log-level
      info
      --config
      /config/probe/config.yaml
      --components-path
      /config/probe/components
    State:          Running
      Started:      Mon, 29 May 2023 19:15:49 +0800
    Ready:          True
    Restart Count:  0
    Readiness:      exec [curl -X POST --max-time 1 --fail-with-body --silent -H Content-ComponentDefRef: application/json http://localhost:3501/v1.0/bindings/mysql -d {"operation": "checkRole", "metadata":{"sql":""}}] delay=0s timeout=1s period=1s #success=1 #failure=2
    Startup:        tcp-socket :3501 delay=0s timeout=1s period=10s #success=1 #failure=3
    Environment Variables from:
      ltest-backup-mysql-env  ConfigMap  Optional: false
    Environment:
      KB_POD_NAME:                ltest-backup-mysql-0 (v1:metadata.name)
      KB_POD_UID:                  (v1:metadata.uid)
      KB_NAMESPACE:               default (v1:metadata.namespace)
      KB_SA_NAME:                  (v1:spec.serviceAccountName)
      KB_NODENAME:                 (v1:spec.nodeName)
      KB_HOST_IP:                  (v1:status.hostIP)
      KB_POD_IP:                   (v1:status.podIP)
      KB_POD_IPS:                  (v1:status.podIPs)
      KB_HOSTIP:                   (v1:status.hostIP)
      KB_PODIP:                    (v1:status.podIP)
      KB_PODIPS:                   (v1:status.podIPs)
      KB_CLUSTER_NAME:            ltest-backup
      KB_COMP_NAME:               mysql
      KB_CLUSTER_COMP_NAME:       ltest-backup-mysql
      KB_CLUSTER_UID_POSTFIX_8:   8e75816b
      KB_POD_FQDN:                $(KB_POD_NAME).$(KB_CLUSTER_COMP_NAME)-headless.$(KB_NAMESPACE).svc
      KB_SERVICE_USER:            <set to the key 'username' in secret 'ltest-backup-conn-credential'>  Optional: false
      KB_SERVICE_PASSWORD:        <set to the key 'password' in secret 'ltest-backup-conn-credential'>  Optional: false
      KB_SERVICE_PORT:            3306
      KB_SERVICE_ROLES:           {"follower":"Readonly","leader":"ReadWrite","learner":"Readonly"}
      KB_SERVICE_CHARACTER_TYPE:  mysql
    Mounts:
      /var/run/secrets/kubernetes.io/serviceaccount from kube-api-access-pdc48 (ro)
  config-manager:
    Container ID:  containerd://3faffcfdc225bbc5795242c9558494a4f27d99ec7e6220f57ef3ff40603eca3f
    Image:         registry.cn-hangzhou.aliyuncs.com/apecloud/kubeblocks-tools:0.6.0-alpha.5
    Image ID:      registry.cn-hangzhou.aliyuncs.com/apecloud/kubeblocks-tools@sha256:b7c3b927862f0c1cb327f8b4a8eaf62a5da06ffe30caade6913a3159940f63c9
    Port:          <none>
    Host Port:     <none>
    Command:
      /bin/reloader
    Args:
      --operator-update-enable
      --log-level
      info
      --tcp
      9901
      --notify-type
      tpl
      --tpl-config
      /opt/config/reload/reload.yaml
    State:          Running
      Started:      Mon, 29 May 2023 19:15:50 +0800
    Ready:          True
    Restart Count:  0
    Environment:
      CONFIG_MANAGER_POD_IP:   (v1:status.podIP)
      DB_TYPE:                mysql
      MYSQL_USER:             <set to the key 'username' in secret 'ltest-backup-conn-credential'>  Optional: false
      MYSQL_PASSWORD:         <set to the key 'password' in secret 'ltest-backup-conn-credential'>  Optional: false
      DATA_SOURCE_NAME:       $(MYSQL_USER):$(MYSQL_PASSWORD)@(localhost:3306)/
    Mounts:
      /opt/config/reload from reload-manager-reload (rw)
      /opt/mysql from mysql-config (rw)
      /var/run/secrets/kubernetes.io/serviceaccount from kube-api-access-pdc48 (ro)
Conditions:
  Type              Status
  Initialized       True
  Ready             True
  ContainersReady   True
  PodScheduled      True
Volumes:
  data:
    Type:       PersistentVolumeClaim (a reference to a PersistentVolumeClaim in the same namespace)
    ClaimName:  data-ltest-backup-mysql-0
    ReadOnly:   false
  annotations:
    Type:  DownwardAPI (a volume populated by information about the pod)
    Items:
      metadata.annotations['cs.apps.kubeblocks.io/leader'] -> leader
      metadata.annotations['apps.kubeblocks.io/component-replicas'] -> component-replicas
  scripts:
    Type:      ConfigMap (a volume populated by a ConfigMap)
    Name:      ltest-backup-mysql-apecloud-mysql-scripts
    Optional:  false
  mysql-config:
    Type:      ConfigMap (a volume populated by a ConfigMap)
    Name:      ltest-backup-mysql-mysql-consensusset-config
    Optional:  false
  reload-manager-reload:
    Type:      ConfigMap (a volume populated by a ConfigMap)
    Name:      mysql-reload-script-ltest-backup
    Optional:  false
  kube-api-access-pdc48:
    Type:                    Projected (a volume that contains injected data from multiple sources)
    TokenExpirationSeconds:  3607
    ConfigMapName:           kube-root-ca.crt
    ConfigMapOptional:       <nil>
    DownwardAPI:             true
QoS Class:                   Burstable
Node-Selectors:              <none>
Tolerations:                 kb-data=true:NoSchedule
                             node.kubernetes.io/not-ready:NoExecute op=Exists for 300s
                             node.kubernetes.io/unreachable:NoExecute op=Exists for 300s
Events:
  Type     Reason                  Age                   From                     Message
  ----     ------                  ----                  ----                     -------
  Normal   Scheduled               32m                   default-scheduler        Successfully assigned default/ltest-backup-mysql-0 to k3d-kb-playground-server-0
  Normal   SuccessfulAttachVolume  32m                   attachdetach-controller  AttachVolume.Attach succeeded for volume "pvc-75236bdf-e0be-490c-9877-aeca57b2306c"
  Normal   Pulled                  32m                   kubelet                  Container image "registry.cn-hangzhou.aliyuncs.com/apecloud/mysqld-exporter:0.14.1" already present on machine
  Normal   Created                 32m                   kubelet                  Created container metrics
  Normal   Started                 32m                   kubelet                  Started container metrics
  Normal   Pulled                  32m                   kubelet                  Container image "registry.cn-hangzhou.aliyuncs.com/apecloud/kubeblocks-tools:0.6.0-alpha.5" already present on machine
  Normal   Created                 32m                   kubelet                  Created container kb-checkrole
  Normal   Started                 32m                   kubelet                  Started container kb-checkrole
  Normal   Pulled                  32m                   kubelet                  Container image "registry.cn-hangzhou.aliyuncs.com/apecloud/kubeblocks-tools:0.6.0-alpha.5" already present on machine
  Normal   Created                 32m                   kubelet                  Created container config-manager
  Normal   Started                 32m                   kubelet                  Started container config-manager
  Normal   Pulled                  19m (x4 over 32m)     kubelet                  Container image "registry.cn-hangzhou.aliyuncs.com/apecloud/apecloud-mysql-server:8.0.30-5.alpha8.20230523.g3e93ae7.7" already present on machine
  Normal   Created                 19m (x4 over 32m)     kubelet                  Created container mysql
  Normal   Started                 19m (x4 over 32m)     kubelet                  Started container mysql
  Warning  BackOff                 5m43s (x15 over 24m)  kubelet                  Back-off restarting failed container
```
```
k logs ltest-backup-mysql-0
Defaulted container "mysql" out of: mysql, metrics, kb-checkrole, config-manager
2023-05-29 11:45:26+00:00 [Note] [MySQL]: Entrypoint script for MySQL Server  started.
2023-05-29 11:45:26+00:00 [Note] [MySQL]: KB_EMBEDDED_WESQL=1
2023-05-29 11:45:26+00:00 [Note] [KubeBlocks]:
2023-05-29 11:45:26+00:00 [Note] [KubeBlocks]: Welcome to the KubeBlocks MySQL Cluster Container
2023-05-29 11:45:26+00:00 [Note] [KubeBlocks]:
2023-05-29 11:45:26+00:00 [Note] [KubeBlocks]: KB_VOLUME_DIR=/data/mysql
2023-05-29 11:45:26+00:00 [Note] [KubeBlocks]: KB_DATA_DIR=/data/mysql/data
2023-05-29 11:45:27+00:00 [Note] [KubeBlocks]: KB_LOG_DIR=/data/mysql/log
2023-05-29 11:45:27+00:00 [Note] [KubeBlocks]: KB_CLUSTER_LEADER=
2023-05-29 11:45:27+00:00 [Note] [KubeBlocks]: KB_CLUSTER_FOLLOWERS=
2023-05-29 11:45:27+00:00 [Note] [KubeBlocks]: KB_CLUSTER_UID=04bbda95-4b0e-4fb4-a1ab-2c258e75816b
2023-05-29 11:45:27+00:00 [Note] [KubeBlocks]: KB_CLUSTER_UID_PATH=/data/mysql/.kb_cluster_uid
2023-05-29 11:45:27+00:00 [Note] [KubeBlocks]: KB_MYSQL_CONF_FILE=/opt/mysql/my.cnf
2023-05-29 11:45:27+00:00 [Note] [KubeBlocks]: KB_MYSQL_PORT=3306
2023-05-29 11:45:27+00:00 [Note] [KubeBlocks]: KB_POD_NAME=ltest-backup-mysql-0
2023-05-29 11:45:27+00:00 [Note] [KubeBlocks]: KB_POD_N=1
2023-05-29 11:45:27+00:00 [Note] [KubeBlocks]: KB_POD_INDEX=0
2023-05-29 11:45:27+00:00 [Note] [KubeBlocks]: KB_POD_HOST=ltest-backup-mysql-0.ltest-backup-mysql-headless
2023-05-29 11:45:27+00:00 [Note] [KubeBlocks]:
2023-05-29 11:45:27+00:00 [Note] [KubeBlocks]:
2023-05-29 11:45:27+00:00 [Note] [KubeBlocks]: CLUSTER_INFO=ltest-backup-mysql-0.ltest-backup-mysql-headless:13306@1
2023-05-29 11:45:27+00:00 [Note] [KubeBlocks]: CLUSTER_LEADER_HOST=
2023-05-29 11:45:27+00:00 [Note] [KubeBlocks]:
2023-05-29 11:45:27+00:00 [Note] [KubeBlocks]: MYSQL_EXTRA_CONF_FILE=/opt/mysql/my.cnf
2023-05-29 11:45:27+00:00 [Note] [KubeBlocks]:
2023-05-29 11:45:27+00:00 [Note] [KubeBlocks]: Injecting configuration /opt/mysql/my.cnf to /etc/my.cnf
2023-05-29 11:45:27+00:00 [Note] [KubeBlocks]: Injecting configuration /etc/mysql/wesql_cluster.cnf to /etc/my.cnf
2023-05-29 11:45:30+00:00 [Note] [KubeBlocks]: DATADIR=/data/mysql/data/
2023-05-29 11:45:30+00:00 [Note] [KubeBlocks]: PID_FILE=/var/run/mysqld/mysqld.pid
2023-05-29 11:45:31+00:00 [Note] [KubeBlocks]: SOCKET=/var/run/mysqld/mysqld.sock
2023-05-29 11:45:34+00:00 [Note] [KubeBlocks]: Switching to dedicated user 'mysql'
2023-05-29 11:45:34+00:00 [Note] [MySQL]: Entrypoint script for MySQL Server  started.
2023-05-29 11:45:34+00:00 [Note] [MySQL]: KB_EMBEDDED_WESQL=1
2023-05-29 11:45:35+00:00 [Note] [KubeBlocks]: DATADIR=/data/mysql/data/
2023-05-29 11:45:35+00:00 [Note] [KubeBlocks]: PID_FILE=/var/run/mysqld/mysqld.pid
2023-05-29 11:45:36+00:00 [Note] [KubeBlocks]: SOCKET=/var/run/mysqld/mysqld.sock
2023-05-29 11:45:38+00:00 [Note] [KubeBlocks]: JEMALLOC_ENABLED=1
2023-05-29 11:45:39+00:00 [Note] [KubeBlocks]: Using jemalloc: LD_PRELOAD=/usr/local/lib/libjemalloc.so.2
2023-05-29 11:45:39+00:00 [Note] [KubeBlocks]: Using persisted data
2023-05-29 11:45:40+00:00 [Note] [KubeBlocks]: Found restore file: /data/mysql/data//.restore_new_cluster
2023-05-29 11:45:40+00:00 [Note] [KubeBlocks]: Initializing consensus meta: 'mysqld --cluster-recover-new-cluster --cluster-recover-snapshot --cluster-info=ltest-backup-mysql-0.ltest-backup-mysql-headless:13306@1 --cluster-id=1 --cluster-start-index=1 --cluster-force-reset-meta=ON --upgrade-wesql'
2023-05-29 11:45:51+00:00 [Note] [KubeBlocks]: Start temporary server in background with backup/snapshot recovery
2023-05-29 11:45:51+00:00 [Note] [KubeBlocks]: Starting temporary server in background: 'mysqld --skip-grant-tables --cluster-recover-new-cluster --cluster-recover-snapshot'
2023-05-29 11:45:51+00:00 [Note] [KubeBlocks]: Waiting for server startup
2023-05-29 11:45:52+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:45:53+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:45:55+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:45:56+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:45:57+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:45:58+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:46:00+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:46:01+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:46:02+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:46:03+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:46:04+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:46:05+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:46:06+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:46:07+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:46:08+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:46:09+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:46:11+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:46:12+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:46:13+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:46:14+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:46:15+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:46:16+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:46:17+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:46:18+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:46:19+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:46:21+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:46:22+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:46:23+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:46:24+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:46:25+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:46:26+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:46:27+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:46:28+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:46:29+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:46:30+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:46:32+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:46:33+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:46:34+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:46:35+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:46:36+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:46:37+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:46:38+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:46:39+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:46:40+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:46:42+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:46:43+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:46:44+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:46:45+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:46:46+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:46:47+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:46:48+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:46:49+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:46:50+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:46:51+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:46:53+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:46:54+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:46:55+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:46:56+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:46:57+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:46:58+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:46:59+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:47:00+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:47:01+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:47:02+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:47:04+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:47:05+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:47:06+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:47:07+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:47:08+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:47:09+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:47:10+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:47:11+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:47:12+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:47:13+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:47:15+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:47:16+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:47:17+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:47:18+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:47:19+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:47:20+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:47:21+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:47:22+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:47:23+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:47:24+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:47:25+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:47:27+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:47:28+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:47:29+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:47:30+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:47:31+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:47:32+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:47:33+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:47:34+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:47:35+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:47:37+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:47:38+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:47:39+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:47:40+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:47:41+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:47:42+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:47:43+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:47:44+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:47:45+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:47:46+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:47:48+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:47:49+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:47:50+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:47:51+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:47:52+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:47:53+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:47:54+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:47:55+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:47:56+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:47:57+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:47:59+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:48:00+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:48:01+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:48:02+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:48:03+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:48:04+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:48:05+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:48:06+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:48:07+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:48:09+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:48:10+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:48:11+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:48:12+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:48:13+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:48:14+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:48:15+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:48:16+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:48:17+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:48:18+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:48:20+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:48:21+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:48:22+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:48:23+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:48:24+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:48:25+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:48:26+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:48:27+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:48:28+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:48:30+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:48:31+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:48:32+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:48:33+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:48:34+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:48:35+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:48:36+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:48:37+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:48:38+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:48:39+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:48:41+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:48:42+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:48:43+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:48:44+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:48:45+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:48:46+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:48:47+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:48:48+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:48:49+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:48:50+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:48:52+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:48:53+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:48:54+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:48:55+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:48:56+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:48:57+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:48:58+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:48:59+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:49:00+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:49:01+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:49:03+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:49:04+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:49:05+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:49:06+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:49:07+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:49:08+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:49:09+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:49:10+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:49:11+00:00 [Warn] [KubeBlocks]: The temporary server is still not started.
2023-05-29 11:49:12+00:00 [ERROR] [KubeBlocks]: Unable to start server.
```

--- a/deploy/apecloud-mysql-scale/values.yaml
+++ b/deploy/apecloud-mysql-scale/values.yaml
@@ -7,7 +7,7 @@ image:
   repository: apecloud/apecloud-mysql-server
   pullPolicy: IfNotPresent
   # Overrides the image tag whose default is the chart appVersion.
-  tag: 8.0.30-5.alpha8.20230523.g3e93ae7.7
+  tag: 8.0.30-5.alpha8.20230523.g3e93ae7.8
 
 ## MySQL Cluster parameters
 cluster:

--- a/deploy/apecloud-mysql/values.yaml
+++ b/deploy/apecloud-mysql/values.yaml
@@ -7,7 +7,7 @@ image:
   repository: apecloud/apecloud-mysql-server
   pullPolicy: IfNotPresent
   # Overrides the image tag whose default is the chart appVersion.
-  tag: 8.0.30-5.alpha8.20230523.g3e93ae7.7
+  tag: 8.0.30-5.alpha8.20230523.g3e93ae7.8
 
 ## MySQL Cluster parameters
 cluster:

